generators/app/index.js

```diff
@@ -134,7 +136,7 @@ module.exports = yeoman.Base.extend({
             jhipsterFunc.addAdminElementTranslationKey('elasticsearch-reindex', 'Reindex Elasticsearch', this.nativeLanguage);
           }
         }
-      } else if (this.clientFramework === 'angular2') {
+      } else if (this.clientFramework === 'angularX') {
```

Comment:
You should make sure it still checks for `angular2` here. I missed it in my previous comment.